src/fsharp/service/service.fs

```diff
@@ -3175,6 +3191,11 @@ type FSharpChecker(legacyReferenceResolver, projectCacheSize, keepAssemblyConten
         let userOpName = defaultArg userOpName "Unknown"
         backgroundCompiler.InvalidateConfiguration(options, startBackgroundCompile, userOpName)
 
+    /// Invalidate a project. Clears all caches related to this project, including check files, stale check files, etc.
```

Comment:
Nit: "this project" --> "the project represented by the given FSharpProjectOptions"